onnxscript/optimizer/_function_folding_test.py

```diff
@@ -138,14 +133,13 @@ def test_multi_users_function_is_not_modified_inplace_after_folding(self):
         """
         )
         optimized = optimizer.optimize(
-            model,
-            onnx_shape_inference=False,
-            num_iterations=1,
+            model, onnx_shape_inference=False, num_iterations=1, inline=False
         )
         self.assertEqual(len(optimized.functions), 2)
         self.assertNotEqual(optimized.functions[0].name, "fun1")
         self.assertNotEqual(optimized.functions[1].name, "fun1")
 
+    @unittest.expectedFailure  # FIXME
```

Comment:
Error in test case model. Imported opset in main graph not same as in function (21 vs 18) ... was being ignored by previous inliner, caught by current inliner.